packages/trimerge-sync-indexed-db/src/trimerge-indexed-db.ts

```diff
@@ -39,6 +38,45 @@ function toSyncNumber(syncId: string | undefined): number {
   return syncId === undefined ? 0 : parseInt(syncId, 36);
 }
 
+/** This creates a native browser BroadcastChannel implementation of EventChannel */
+function getBroadcastChannelEventChannel<CommitMetadata, Delta, Presence>(
```

Comment:
pull this and EventChannel out to a second file

Reply:
unfortunately, trimerge-sync doesn't support dom types so I've split them into two separate files.